spec/javascripts/components/short_form_language_switcher_spec.coffee

```diff
@@ -0,0 +1,27 @@
+do ->
```

Comment:
@kaptron do you notice any weird syntax errors here? I just noticed when running the tests that this file wasn't picked up.